src/mgmt/srv_drpc.c

```diff
@@ -1731,89 +1731,123 @@ pool_query_free_tier_stats(Mgmt__PoolQueryResp *resp)
 	resp->n_tier_stats = 0;
 }
 
+static int
+rank_list_to_str(d_rank_list_t *ranks, char **ranks_str)
```

Comment:
Why are we not using `d_rank_list_to_str`?

Reply:
- [ ] Remove useless rank_list_to_str() function
- [ ] Add unit test to d_rank_list_str() function

Reply:
> * [x]  Remove useless rank_list_to_str() function
> * [x]  Add unit test to d_rank_list_str() function

Fixed with commit 4d95fbe5c16f7c44ee969ac5ac869797fa5b3429